spec/models/rook_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    let(:game) {create :game}
+
+    it "should properly evaluate malformed moves that are otherwise valid" do
+      white_rook = game.pieces.find_by(rank: 1, file: 1)
+      black_rook = game.pieces.find_by(rank: 8, file: 1)
+      expect(white_rook.valid_move?(0, 1)).to eq(false)
+      expect(white_rook.valid_move?(1, 0)).to eq(false)
+      expect(white_rook.valid_move?(0, 0)).to eq(false)
+      expect(white_rook.valid_move?(0, 2)).to eq(false)
+      expect(white_rook.valid_move?(2, 0)).to eq(false)
+      expect(white_rook.valid_move?(-8, -9)).to eq(false)
+      expect(white_rook.valid_move?(-1, -9)).to eq(false)
+      expect(white_rook.valid_move?(1, -8)).to eq(false)
+      expect(white_rook.valid_move?(1, 1)).to eq(false)
+      expect(black_rook.valid_move?(7, 0)).to eq(false)
+      expect(black_rook.valid_move?(8, 0)).to eq(false)
+      expect(black_rook.valid_move?(9, 0)).to eq(false)
+      expect(black_rook.valid_move?(9, 1)).to eq(false)
+      expect(black_rook.valid_move?(9, 2)).to eq(false)
+      expect(black_rook.valid_move?(8, 1)).to eq(false)
+      expect(black_rook.valid_move?(15, -9)).to eq(false)
+      expect(black_rook.valid_move?(9, -9)).to eq(false)
+      expect(black_rook.valid_move?(8, -8)).to eq(false)
+    end
```

Comment:
The `expect` statements that you added here don't actually cover the scenarios that I mentioned, all of which involve files greater than 8. Here they are again for reference:

- `rank > 8 && file > 8`
- `rank < 1 && file > 8`
- A valid rank and `file > 8`

Basically, I'd like to cover the eight directions in which you can go outside the board (two vertical, two horizontal, and four diagonal).